packages/garbo/src/mood.ts

```diff
@@ -227,23 +228,29 @@ export function useBuffExtenders(): void {
   }
 }
 
-const stings = [
-  ...$effects`Apoplectic with Rage, Barfpits, Berry Thorny, Biologically Shocked, Bone Homie, Boner Battalion, Coal-Powered, Curse of the Black Pearl Onion, Dizzy with Rage, Drenched With Filth, EVISCERATE!, Fangs and Pangs, Frigidalmatian, Gummi Badass, Haiku State of Mind, It's Electric!, Jabañero Saucesphere, Jalapeño Saucesphere, Little Mouse Skull Buddy, Long Live GORF, Mayeaugh, Permanent Halloween, Psalm of Pointiness, Pygmy Drinking Buddy, Quivering with Rage, Scarysauce, Skeletal Cleric, Skeletal Rogue, Skeletal Warrior, Skeletal Wizard, Smokin', Soul Funk, Spiky Frozen Hair, Stinkybeard, Stuck-Up Hair, Can Has Cyborger, Feeling Nervous`,
-  $effect`Burning, Man`,
-  $effect`Yes, Can Haz`,
-];
-const textAlteringEffects = $effects`Can Has Cyborger, Dis Abled, Haiku State of Mind, Just the Best Anapests, O Hai!, Robocamo`;
-export const teleportEffects = $effects`Teleportitis, Feeling Lost, Funday!`;
-const otherwiseBadEffects = $effects`Temporary Blindness`;
+const damageEffects = Effect.all().filter(
+  (x) =>
+    numericModifier(x, "Thorns") ||
+    numericModifier(x, "Sporadic Thorns") ||
+    numericModifier(x, "Damage Aura") ||
+    numericModifier(x, "Sporadic Damage Aura"),
+);
```

Comment:
```suggestion
const damageEffects = Effect.all().filter(
  (x) =>["Thorns", "Sporadic Thorns", "Damage Aura", "Sporadic Damage Aura"].some((modifier) => numericModifier(x, modifier) > 0));
```

Reply:
I would pull out the array of strings from being defined within the filter.